SeaPublicWebsite/Views/EnergyEfficiency/AnswerSummary.cshtml

```diff
@@ -696,12 +696,19 @@
         {
             Rows = rows
         }))
-
-        <a href="@Url.Action("YourRecommendations_Get", "EnergyEfficiency", new {reference = Model.UserDataModel.Reference})" role="button" draggable="false" class="govuk-button govuk-!-margin-top-2 govuk-!-margin-bottom-8 govuk-button--start" data-module="govuk-button">
-            See results
-            <svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19" viewBox="0 0 33 40" aria-hidden="true" focusable="false">
-                <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z"/>
-            </svg>
-        </a>
+        
+        <form action="@Url.Action("AnswerSummary_Post", "EnergyEfficiency", new { reference = Model.UserDataModel.Reference })" method="post" novalidate>
```

Comment:
Created a form so this goes to a post request. Can I do that with just an <a> link?

Reply:
I think it makes sense for this to be a post as it is submitting the final answer data and getting the recomendations.